debian/libnymea1-dev.install.in

```diff
@@ -1,4 +1,12 @@
 usr/lib/@DEB_HOST_MULTIARCH@/libnymea.so
-usr/include/nymea/* usr/include/nymea
-libnymea/integrations/plugin.pri usr/include/nymea/
+usr/include/nymea/*.h
+usr/include/nymea/coap/*
+usr/include/nymea/experiences/*
+usr/include/nymea/hardware/*
+usr/include/nymea/integrations/*
+usr/include/nymea/jsonrpc/*
+usr/include/nymea/network/*
+usr/include/nymea/platform/*
+usr/include/nymea/time/*
+usr/include/nymea/types/*
```

Comment:
Shouldn't those be installed by qmake for the dev package any more?

Reply:
Yes, the issue was that the plugin.pri is also in /usr/include/nymea/. If I just select /usr/include/nymea/* in the libnymea-dev, it will also have the plugin.pri, but for $reasons I needed to move the pri into the nymea-sdk-tools package

reasons being: you can link against libnymea without the need of the plugin.pri, but you can't use the nymea-plugininfocompiler without the pri. I first tried with having a dependency from nymea-dev-tools to libnymea1-dev, but that would break on xenial in a cross-compile scenario... Also, thinking about it twice, the .pri actually does belong to the dev-tools package anyways... makes the install.in files a bit more ugly tho...

Reply:
understood